src/components/ButtonPanel.js

```diff
@@ -1,43 +1,46 @@
 import PropTypes from 'prop-types';
 import Button from './Button';
+import './ButtonPanel.css';
 
 const ButtonPanel = props => {
   const handleClick = buttonName => {
     props.clickHandler(buttonName);
   };
 
   return (
-    <>
-      <div>
-        <Button name="AC" clickHandler={handleClick} />
-        <Button name="+/-" clickHandler={handleClick} />
-        <Button name="%" clickHandler={handleClick} />
-        <Button name="÷" clickHandler={handleClick} />
-      </div>
-      <div>
-        <Button name="7" clickHandler={handleClick} />
-        <Button name="8" clickHandler={handleClick} />
-        <Button name="9" clickHandler={handleClick} />
-        <Button name="x" clickHandler={handleClick} />
-      </div>
-      <div>
-        <Button name="4" clickHandler={handleClick} />
-        <Button name="5" clickHandler={handleClick} />
-        <Button name="6" clickHandler={handleClick} />
-        <Button name="-" clickHandler={handleClick} />
-      </div>
-      <div>
-        <Button name="1" clickHandler={handleClick} />
-        <Button name="2" clickHandler={handleClick} />
-        <Button name="3" clickHandler={handleClick} />
-        <Button name="+" clickHandler={handleClick} />
-      </div>
-      <div>
-        <Button name="0" clickHandler={handleClick} />
-        <Button name="." clickHandler={handleClick} />
-        <Button name="=" clickHandler={handleClick} />
-      </div>
-    </>
+    <div className="button-panel">
+      <>
+        <div className="row1">
+          <Button name="AC" clickHandler={handleClick} />
+          <Button name="+/-" clickHandler={handleClick} />
+          <Button name="%" clickHandler={handleClick} />
+          <Button name="÷" clickHandler={handleClick} />
+        </div>
+        <div className="row2">
+          <Button name="7" clickHandler={handleClick} />
+          <Button name="8" clickHandler={handleClick} />
+          <Button name="9" clickHandler={handleClick} />
+          <Button name="x" clickHandler={handleClick} />
+        </div>
+        <div className="row3">
+          <Button name="4" clickHandler={handleClick} />
+          <Button name="5" clickHandler={handleClick} />
+          <Button name="6" clickHandler={handleClick} />
+          <Button name="-" clickHandler={handleClick} />
+        </div>
+        <div className="row4">
+          <Button name="1" clickHandler={handleClick} />
+          <Button name="2" clickHandler={handleClick} />
+          <Button name="3" clickHandler={handleClick} />
+          <Button name="+" clickHandler={handleClick} />
+        </div>
+        <div className="row5">
+          <Button name="0" className="button-0" clickHandler={handleClick} />
```

Comment:
### [Optional]

While we focus on testing in this project, we can also make your project better.
Kindly give the `button 0` 50% `width` instead `25%`.

**With 25% width which is your currently width;**
![image](https://user-images.githubusercontent.com/31897434/110413287-e2474e00-808d-11eb-9642-70dcbb2fdc59.png)

**With 50% width;**
![image](https://user-images.githubusercontent.com/31897434/110413354-0014b300-808e-11eb-8a00-175f5f574405.png)